integration_tests/array_03.py

```diff
@@ -1,20 +1,20 @@
-from lpython import Allocatable, f64, i32
+from lpython import Allocatable, f64, i32, Const
 from numpy import empty, float64, int32
 
 def f():
-    n: i32 = 5
+    n: Const[i32] = 5
     a: Allocatable[f64[:]] = empty((n,), dtype=float64)
     i: i32
     for i in range(n):
         a[i] = f64(i+1)
     for i in range(n):
         assert abs(a[i] - f64(i + 1)) < 1e-12
-    b: Allocatable[i32[:]]
-    n = 10
-    b = empty((n,), dtype=int32)
-    for i in range(n):
+
+    m: Const[i32] = 10
+    b: Allocatable[i32[:]] = empty((m,), dtype=int32)
```

Comment:
Here `m` doesn't need to be constant. Instead this test is testing that it can be a runtime variable. So we need to preserve the test.